streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -387,6 +387,28 @@ public synchronized V delete(K key) {
         return rocksDbIterator;
     }
 
+    public synchronized KeyValue<K, V> first() {
```

Comment:
tests for the `first()` and `last()` methods.

Reply:
See my other comment above on `RocksDBSegmentedBytesStore#all()`, we can get rid of these two functions as well (note the RocksIterator.seeks are quite expensive so we should avoid at best).